chef/cookbooks/crowbar/templates/default/crowbar-router-migration.sh.erb

```diff
@@ -35,17 +35,23 @@ set +x
 source /root/.openrc
 set -x
 
+if ssh $hostname "test -e $UPGRADEDIR/crowbar-router-migrated" ; then
+    log "l3 agent was already evacuated from host $hostname"
+    touch $UPGRADEDIR/crowbar-router-migration-ok
+    exit 0
+fi
```

Comment:
Yes, I did.
I was not sure which way it is better, but here it at least felt correct to create that variant of -ok file (`crowbar-router-migrated`) as it seems kind of consistent with other scripts. And if creating of file is here, why not checking for it's presence... (it was already established that we have ssh access to other nodes of the cluster)